site/public/index.php

```diff
@@ -45,7 +45,7 @@
  */
 function exception_handler($throwable) {
     global $core;
-    $message = ExceptionHandler::handleException($throwable);
+    $message = ExceptionHandler::handleException($throwable, $core->getUserId());
```

Comment:
Why not just call `getUser()->getId()`? If it's the null case, then could do a check or pass just the user then do the null check in the function instead. Don't see much of a point in adding another `Core.php` function for this case.

Reply:
Changed, but why do we not have a getUserId method in general? In this baseline there is a line where userId is written but never read. Do we want to read the variable or is `getUser()->getId()` better?

Reply:
I'd be in favor of just removing `user_id` from `Core.php` as it has no use and `getUser()` provides us with what we need already. If you want to do that, please do it in a separate PR.

Reply:
sounds good